airtime_mvc/application/views/scripts/index/index.phtml

```diff
@@ -57,7 +57,7 @@
         <div id="tab-4" class="podcast tab_content">
             <div class="rss-title-wrapper">
                 <div id="rss-subscribe">
-                    <a target="_blank" href="<?php echo $this->stationPodcastRssUrl ?>" class="rss_icon"><span><?php echo _("Subscribe") ?></span><img src="css/radio-page/img/rss.png"></a>
+                    <a target="_blank" href="<?php echo $this->stationPodcastRssUrl ?>" class="rss_icon"><span>Subscribe </span><img src="css/radio-page/img/rss.png"></a>
```

Comment:
Why did you remove the translation call here?

Reply:
Unintended.

Fixed in https://github.com/LibreTime/libretime/pull/1119/commits/b8f4a3392716b5be2ab89139cc5b65b973dec523.